packages/core/src/Modules/Cashier/Containers/on-ramp.jsx

```diff
@@ -10,20 +10,52 @@ import 'Sass/app/modules/on-ramp.scss';
 const OnRamp = ({
     is_onramp_modal_open,
     filtered_onramp_providers,
+    menu_options,
     onramp_popup_modal_title,
     setIsOnRampModalOpen,
     should_show_dialog,
     resetPopup,
     onMount,
     onUnmount,
+    history,
 }) => {
+    const [selected_cashier_path, updateSelectedChaierPath] = React.useState('/cashier/on-ramp');
+    React.useEffect(() => {
+        if (menu_options?.length) {
+            history.push(selected_cashier_path);
+        }
+    }, [selected_cashier_path]);
     React.useEffect(() => {
         onMount();
         return () => onUnmount();
     }, []);
-
+    const getActivePaths = () => {
+        const items = [];
+        if (menu_options?.length) {
+            menu_options.forEach(menu_item => {
+                const item = {};
+                item.text = localize(menu_item.label);
```

Comment:
This won't work, `localize` will send `menu_item.label` to Crowdin, please `localize` at the source string.

Reply:
Since we've get this from routes-config we don't need to localize it